### PR TITLE
Front min update

### PR DESCRIPTION
**Pull request by razhnoff**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,26 @@
...
+.pnp.js
+
+# testing
+/coverage
+
+# idea
 .idea/
+
+# production
+/build
```
Comment (PavloKovalov) [inline .gitignore]:
```diff
- /build
+ build/
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    "react": "^16.12.0",
+    "react-calendar": "^2.19.2",
+    "react-dom": "^16.12.0",
+    "react-leaflet": "^2.6.1",
+    "react-router": "^5.1.2",
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "^3.3.0",
+    "react-text-mask": "^5.4.3",
+    "react-yandex-maps": "^3.1.0",
+    "reveal": "0.0.4",
```
Comment (dasnein) [inline package.json]: Check your npm packages list. There are some unused packages, e.g. **reveal**, **draft-convert**, **axios-mock-adapter**

Comment (razhnoff) [reply package.json]: @DasneiN update package dependecies and remove useless, axios-mock-adapter will be used for fetching mockable data.

**Review by PavloKovalov — APPROVED** [review]
Seems legit 👮🏻‍♂️

**Pull request merged**